doc/source/whatsnew/v0.16.1.txt

```diff
@@ -55,7 +55,7 @@ Bug Fixes
 
 
 - Bug in ``transform`` causing length mismatch when null entries were present and a fast aggregator was being used (:issue:`9697`)
-
+- Fixed bug in groupby.apply() that would cause error if no values returned. (:issue:`9685`)
```

Comment:
use backticks around ``groupby.apply`` and reprase as bit: "would raise if the return value of the user passed function was ``None`` "